docs/general/polkadot-v1.md

```diff
@@ -0,0 +1,154 @@
+  validators there are securing the network (i.e. how easy it is to collude validators and stop the
+  network). Additionally, it is also important to look at how many validators are owned by single
+  operators (this will determine the degree of social centralization of the network), and how many
+  validators run on the same service provider (this will determine the degree of physical
+  centralization of the network).
+- **flexibility**, intended as how flexible the blockspace is, what can be done with it, and what
+  type of data can be stored. Data quality plays an important role depending on the type of network.
+  One might avoid having situations in which poor quality data flood blockspace hindering the prompt
+  execution of vital processes.
+- **availability**, intended as how available blockspace is and how difficult it is to access it. It
+  should not be too difficult to get your hands on it so that any business model (not only DeFi) can
+  thrive using it. Ideally, a marketplace must drive the blockspace price based on demand, with
+  secondary market options to ensure the usage of "second-hand" blockspace.
+
+Polkadot's has been designed around those core blockspace principles. However, its design can be
+further improved. Tasks such as balances transfers, staking, governance, etc. can be delegated to
+[system parachains](../learn/learn-system-chains.md) to increase flexibility and to focus the use of
+the relay-chain to provide shared security and interoperability. Blockspace is only accessible
+through slot auctions, but an auction winner has access to a "freighter of blocks" regardless it is
+needed or not. This creates high entry barriers and it can lead to waste of energy and resources.
+
+For more information about blockspace the [this interview](https://youtu.be/e1vISppPwe4) to Robert
+Habermeier as well as [this article](https://www.rob.tech/polkadot-blockspace-over-blockchains/) by
+him.
+
+### A Perspective Shift
+
+As with many other projects before Polkadot, at some point in time after achieving the
+initially-planned goals, a perspective shift allows you to understand better what your project is
+about and what you actually have built. This allows you to "run the extra mile" and achieve more
+than what was originally planned.
+
+The quote below by [Marcel Proust](https://en.wikipedia.org/wiki/Marcel_Proust) must remind us that
+sometimes a perspective shift is crucial in _understanding_ the world, and perhaps it is more
+important than _seeing more_ of the world.
+
+_The only true voyage of discovery, the only fountain of Eternal Youth, would be not to visit
+strange lands but to possess other eyes._
+
+Thus, if we start to see Polkadot with _other eyes_ we can truly envision its potential and what it
+could become.
+
+Enter Polkadot 2.0, a long-term vision looking at Polkadot as a decentralized, secure, ubiquitous
+computing engine to power the next-generation of Web3 application.
```

Comment:
A small excerpt here could be warranted, let me know what you think. Feel free to modify it, it is very rough.  I want to capture the concepts of "spaces", cores, etc: 

> In the context of Polkadot 2.0, the perspective shift will be crucial to realize the broader ideas it will bring due to its implementation.  From blockchains, a relatively two-dimensional structure, to defining governable, three-dimensional spaces of secure, decentralized computing.  This space would comprise multiple cores, similar to how a modern CPU is structured.  These cores can then communicate using a common language, XCM, which empowers applications built across these cores.   The goal is not a chain-centric future but an application-centric one that provides users true value and utility.